azkaban-executor/azkaban-emr-jobtype/src/main/java/uk/gov/dwp/dataworks/azkaban/services/DelegateService.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.dwp.dataworks.azkaban.services;
+
+import uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class DelegateService {
```

Comment:
Not a good name for what this class does.

Reply:
The class is delegated to by the CompositeService to perform a subset of the required actions. It is a delegate.